ex16v2_ch4.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: state constant
+LEGAL_DRIVING_AGE = 16
+
+#step2: get input from user
+age = int(input("Enter your age: "))
+
+#step3: set up 3 conditionals: if user provides invalid input, if 16 or older, or if younger than 16
+if age < 0:
+    print("Please enter a valid age.")
```

Comment:
The `if` on line 16 is a separate test from the `if-else` combination on lines 18-21, so I'd recommend adding a blank line after line 17 to separate the two.

Reply:
@amymhaddad Apply the comment above and it will be approved.